src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -17,4 +24,62 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     // Forward request to register.jsp file
     request.getRequestDispatcher("/WEB-INF/view/register.jsp").forward(request, response);
   }//doGet
+
```

Comment:
Quick clarification question: Did you type up the code below yourself? I just finished reviewing Tishauna's code, and I think most of this should have already been written in her task? I might be very confused and missing something

Reply:
I think the reason things are inconsistent between my code and Tishauna's is because mine contain the changes from 4 (create register.jsp), whereas hers do not. ( in some places they do?) For this file in particular, I needed to make changes to doPost function, which is not created until Tishauna's part 5. I needed to copy-paste it from her pull request (because they are not merged yet) in order to proceed with 9.

Reply:
I know, its kind of confusing haha. That's why I had to ask during the meeting if I could that.